frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -239,6 +244,26 @@ export const PersistentVolumeClaimsDetailsPage = (props) => (
       navFactory.details(Details),
       navFactory.editYaml(),
       navFactory.events(ResourceEventStream),
+      ...plugins.registry
```

Comment:
```ts
const resourceTabPageExtensions = useExtensions<ResourceTabPage>(isResourceTabPage)
  .filter( /*..*/ )
  .map( /*..*/ )
```

Reply:
Question: it seems that we're consuming the new `ResourceTabPage` extension in a specific context (PVC details page) - shouldn't we support adding tabs to **all** kinds of resource details pages?

If so, the context of this extension should be "all resource details pages".

Reply:
I think it would be better to put it on the details page. I am updating the PR with the extension in the details page. :)